crates/atuin/src/command/client/import.rs

```diff
@@ -26,6 +27,8 @@ pub enum Cmd {
     ZshHistDb,
     /// Import history from the bash history file
     Bash,
+    /// Import history from user csv history file
+    CsvHistoryImporter,
```

Comment:
```suggestion
    Csv,
```

we have `atuin import bash`, `atuin import zsh` etc, so `atuin import zsh` makes more sense here

Reply:
will update